src/app/users/users-achievements/users-achievements.component.html

```diff
@@ -8,7 +8,10 @@
   <mat-toolbar class="primary-color font-size-1">
     <span>{{user.firstName.concat(' ', user.lastName).trim() || user.name}}</span>
     <span class="toolbar-fill"></span>
-    <a mat-raised-button color="accent" i18n [routerLink]="['/myAchievements/update']" *ngIf="ownAchievements"><mat-icon>mode_edit</mat-icon><span>Edit Achievements</span></a>
+    <a mat-raised-button color="accent" i18n [routerLink]="['/myAchievements/update']"><mat-icon>mode_edit</mat-icon>
+		<span *ngIf="!ownAchievements">Add Achievements</span>
+		<span *ngIf="ownAchievements">Edit Achievements</span> 
+	</a>
```

Comment:
Indentation should be at 2 space increments with the spans inside the a element.

Reply:
@paulbert I've added indentation at 2 space increments. I am not sure why it won't render the same way when I push the commit.
 [![indentation](https://user-images.githubusercontent.com/21087353/53031585-bfa85500-3432-11e9-8cf8-35282c197b70.PNG)
](url)

Reply:
Ok, that looks good.  I'll push the commit so we can move it along, but we can troubleshoot this tonight.

Reply:
I found the issue.  Looks like there were some tab characters in the indentation.  Github shows that as one space, my editor shows them as two, and your editor shows them as four.

There should be an option in your editor to set the tab key to insert X number of spaces, so we can try to set that tonight.  Or if you want to try out VS code that will automatically have that set for this repository.